startupscript/butane/probe-user-access.sh

```diff
@@ -29,7 +29,7 @@ if [[ "$(docker container inspect -f '{{.State.Running}}' "${CONTAINER_NAME}")"
     fi
 fi
 
-LOAD="$(awk '{print $1}' /proc/loadavg)" # 1-minute average load
```

Comment:
1 min seems to be too noisy.

Reply:
3 == 15 mins?

Reply:
ya there're three options. 1 min, 5 min and 15 min